src/scheduler/job_info.c

```diff
@@ -4857,3 +4914,96 @@ preemption_similarity(resource_resv *hjob, resource_resv *pjob, schd_error *full
 	return match;
 }
 
+/**
+ * @brief Create the resources_released and Resources_Rel for a job 
```

Comment:
Yeah, there is confusion here. Resources_Rel should be resource_released_list. I've changed the comment to reflect that.